### ISSUE TITLE
Bug: Cannot access rules

### ISSUE THREAD

**Issue by stephanschubert** [csstools/css-has-pseudo#5]
Requesting Google fonts via [`gatsby-plugin-typhography`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-typography) inserts:

```html
<link href="//fonts.googleapis.com/css?family=Source+Serif+Pro:600|Source+Sans+Pro:400,400i,700" rel="stylesheet" type="text/css">
```

which results in the following error:

```
Uncaught (in promise) DOMException: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules
    at walkStyleSheet
```

**Comment by jonathantneal** [comment]
Thanks for reporting this. It has been resolved (I hope) in v0.10.0. Would you give it a try?

As to the solution: The only way I could find one can avoid this is with a try/catch block. See https://github.com/csstools/css-has-pseudo/commit/194f13c2fc603737e6a91a4f8c35ac1d718566c9

**Comment by stephanschubert** [comment]
It works now, thanks. :tada:
